src/main/java/com/SpringBoot/RestApi/UserManagement/UserDetails.java

```diff
@@ -15,18 +15,18 @@ public class UserDetails {
 	
 	@Id
 	@GeneratedValue(strategy=GenerationType.AUTO)
-	
 	@Column(name="user_id")
 	public long userId;
 	
+	@Column(columnDefinition="VARCHAR(25)")
 	public String name;
 
-	@Column(name="mobile_number")
+	@Column(name="mobile_number" ,columnDefinition="VARCHAR(15)")
 	public String mobileNumber;
 	
 	public RoleEnum role;
 	
-	@Column(name="is_deleted")
+	@Column(name="is_deleted" , columnDefinition = "BOOLEAN CHECK (is_deleted IN (true,false))")
```

Comment:
you can set default value as false then you don't need to pass this value while creating new user everytime.

Reply:
there are spring validators also, you can use those as well.
https://reflectoring.io/bean-validation-with-spring-boot/